ivy/functional/backends/paddle/experimental/creation.py

```diff
@@ -16,12 +16,12 @@
 # Code from cephes for i0
 
 
-def _kaiser_window(window_length, beta):
-    n = paddle.arange(0, window_length)
-    alpha = (window_length - 1) / 2.0
-    return paddle_backend.i0(
-        beta * paddle.sqrt(1 - paddle_backend.divide((n - alpha), alpha) ** 2.0)
-    ) / paddle_backend.i0(beta)
+def _kaiser_window(M, beta):
```

Comment:
I couldn't find this in the paddle backend, but it seems that you have followed the naming convention of `numpy`
I personally think we follow the naming convention of `torch` and `tensorflow` 
Let's say for example tensorflow:

```
def kaiser_window(
    window_length: int,
    periodic: bool = True,
    beta: float = 12.0,
    *,
    dtype: Optional[tf.DType] = None,
    out: Optional[Union[tf.Tensor, tf.Variable]] = None,
) -> Union[tf.Tensor, tf.Variable]:
```
Let me know your thoughts @MahmoudAshraf97 :slightly_smiling_face:

Reply:
Thats the auxiliary equation function, if you want I can fit it all into one function but that would be slightly counter intuitive if you consider that we would have to write the same thing again for the bessel derived window if we go with the auxiliary

Reply:
Sorry My bad I did saw the numpy docs and thought this for the paddle backend function you implemented.
Thanks for the head up @Enigmafiend !